src/estimation/include/iDynTree/Estimation/BerdyHelper.h

```diff
@@ -498,14 +521,14 @@ class BerdyHelper
     /**
      * Resize and set to zero Berdy matrices.
      */
-    bool resizeAndZeroBerdyMatrices(MatrixDynSize & D, VectorDynSize & bD,
-                                    MatrixDynSize & Y, VectorDynSize & bY);
+    bool resizeAndZeroBerdyMatrices(SparseMatrix &D, VectorDynSize &bD,
+                                    SparseMatrix &Y, VectorDynSize &bY);
 
     /**
      * Get Berdy matrices
      */
-    bool getBerdyMatrices(MatrixDynSize & D, VectorDynSize & bD,
-                          MatrixDynSize & Y, VectorDynSize & bY);
```

Comment:
I did not realize we were nuking the old interface. For compatibility can we keep also the old interface, with an inefficient conversion from sparse to dense? This would simplify life mainly for my matlab code that depends on this.